### ISSUE TITLE
Include a default set of tags

### ISSUE THREAD

**Issue by justinbarclay** [rougier/svg-tag-mode#11]
I think it would be worthwhile, as a user onboarding experience, for this package to include a default set of tags bindings that they can enable.

One of the use cases I'm thinking of is If someone downloads this from MELPA, they won't have access to example1.el or example2.el to be able to be able to use at their discretion.

**Comment by rougier** [comment]
Make sense. Would you have some suggestiosn based on the examples ? We could sure at least add the TODO, but what else?

**Comment by justinbarclay** [comment]
I think restricting these customizations to Org makes sense, it's the only base install that changing the iconography to highlight specific parts makes sense. So like you I think the TODO keywords are sane defaults. After that tags _could_ be nice, but I don't really use tags that much so I don't have a strong inclination on that. However, I do think priorities could be another nice surprise. I currently use [org-fancy-priorities](https://github.com/harrybournis/org-fancy-priorities) for this.

**Comment by rougier** [comment]
Makes sense. The default would thus be "TODO", "DONE", "#A", "#B", "#C". Coudl you make a PR?

**Comment by japhir** [comment]
Having it depend on `org-todo-keywords` would be even better :)

**Comment by rougier** [comment]
Good point. Do you know how can we extract keywords and priorities text only?

**Comment by rougier** [comment]
@japhir Thanks for the hint. I woudl prefer to avoid parsing this list to only extract the keywords. I guess there might be some way but I need to search.

**Comment by mwsundberg** [comment]
Another good set of defaults might be for key chords in Help buffers. They might be a bit harder to match with regex, yet that was the first use case I thought of after seeing this (and what I plan to try to use it for).

**Comment by rougier** [comment]
By the way, there's now a [svg-lib](https://github.com/rougier/svg-lib) that is available on ELPA. I'm not sure to add the default tags inside, but we could consider a dedicated package for common tags.

**Comment by Kazark** [comment]
I think trying to make this "just work" with existing org stuff would be really cool---I kind of DWIM default setup out of the box. `org-todo-keywords` is definitely a disappointment of a data structure, though. :/

**Comment by rougier** [comment]
So I think solution woudl be to add dedicated examples (e.g. one for org mode)
